FirebaseCombineSwift/Tests/Unit/Auth/AuthMocks.swift

```diff
@@ -0,0 +1,114 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+@testable import FirebaseAuth
+
+let expectationTimeout: TimeInterval = 2
+let kAccessTokenTimeToLive: TimeInterval = 60 * 60
+let kRefreshToken = "REFRESH_TOKEN"
+let kAccessToken = "ACCESS_TOKEN"
+
+class AuthBackendImplementationMock: NSObject, FIRAuthBackendImplementation {}
```

Comment:
Nit: I think these would quality as a `Fake` and not a `Mock` right?

Reply:
Removed them, as they weren't even used.